pkg/parser/libfuzzer/stacktrace/stacktrace.go

```diff
@@ -14,6 +14,10 @@ import (
 var framePattern = regexp.MustCompile(
 	`#(?P<frame_number>\d+)\s+0x[a-fA-F0-9]+\s+in\s+(?P<function>(\(anonymous namespace\))?[^(\s]+).*\s(?P<source_file>\S+?):(?P<line>\d+):?(?P<column>\d*)`)
 
+// Special pattern for Java stack traces which does not filter out java base classes.
+// This has to checked elsewhere since go regex does not allow lookups to filter out specific words.
```

Comment:
I suggest we move the explanation about Java base classes to the code where we actually exclude stack frames of Java base classes (see below)
```suggestion
// Special pattern for Java stack traces
```